lms/djangoapps/commerce/api/v0/views.py

```diff
@@ -146,6 +149,11 @@ def post(self, request, *args, **kwargs):
 
         # Make the API call
         try:
+            # Pass along Sailthru campaign id
+            campaign_cookie = request.COOKIES.get(SAILTHRU_CAMPAIGN_COOKIE)
+            if campaign_cookie:
+                api_session.cookies = requests.utils.cookiejar_from_dict({SAILTHRU_CAMPAIGN_COOKIE: campaign_cookie})
```

Comment:
This overwrites all of the cookies in the session (which is probably ok for now, if that's the only cookie). But maybe it would be safer/clearer to just use `api_session.cookies[SAILTHRU_CAMPAIGN_COOKIE] = campaign_cookie`?